packages/sdk/src/components/sdk-get/sdk-get.js

```diff
@@ -125,7 +125,12 @@ export class SdkGet extends React.Component {
   }
 }
 
-export default connect(
-  null,
-  dispatch => ({ dispatch })
-)(SdkGet);
+export default function createSdkGet(connectOptions = {}) {
```

Comment:
Do we still need this? Disregard comment if so.

Reply:
Not necessarily, but I felt like keeping it as it's mostly an internal change and can already allow consumers of the package to pass a custom context. It's also not documented so we might use it to experiment with things.

However, I can also revert it if we don't want to "give this out" just yet.

Reply:
I personally would not give this out. Even an undocumented breaking change or an public API is a breaking change. If we want to play with something like this maybe we should do so from a canary release not from the latest dist-tag. I am also not sure if we would name the API like that. Just my two cents though.

Reply:
I personally would not give this out. Even an undocumented breaking change or an public API is a breaking change. If we want to play with something like this maybe we should do so from a canary release not from the latest dist-tag. I am also not sure if we would name the API like that. Just my two cents though.